content/id/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -0,0 +1,109 @@
+weight: 10
+card: 
+  name: concepts
+  weight: 40
+---
+
+{{% capture overview %}}
+Laman ini menjelaskan bagaimana objek-objek Kubernetes direpresentasikan di dalam API Kubernetes, 
+dan bagaimana kamu dapat merepresentasikannya di dalam format `.yaml`.
+{{% /capture %}}
+
+{{% capture body %}}
+## Memahami Konsep Objek-Objek yang Ada pada Kubernetes
+
+*Objek-Objek Kubernetes* adalah entitas `persisten` di dalam sistem Kubernetes. 
+Kubernetes menggunakan entitas ini untuk merepresentasikan `state` yang ada pada 
+kluster kamu. Secara spesifik, hal itu dapat dideskripsikan sebagai:
+
+* Aplikasi-aplikasi `containerized` apa sajakah yang sedang dijalankan (serta pada `node` apa aplikasi tersebut dijalankan)
+* `Resource` yang tersedia untuk aplikasi tersebut
+* `Policy` yang mengatur bagaimana aplikasi tersebut dijalankan, misalnya `restart`, `upgrade`, dan `fault-tolerance`.
+
+Objek Kubernetes merupakan sebuah `"record of intent"`--yang mana sekali kamu membuat suatu objek, 
+sistem Kubernetes akan bekerja secara konsisten untuk menjamin 
+bahwa objek tersebut akan selalu ada. Dengan membuat sebuah objek, secara tak langsung kamu 
+memberikan informasi pada sistem Kubernetes mengenai perilaku apakah yang kamu inginkan pada `workload` kluster yang kamu miliki;
+dengan kata lain ini merupakan definisi `state` kluster yang kamu inginkan.
+
+Untuk menggunakan objek-objek Kubernetes--baik membuat, mengubah, atau menghapus objek-objek tersebut--kamu 
+harus menggunakan [API Kubernetes](/docs/concepts/overview/kubernetes-api/). 
+Ketika kamu menggunakan perintah `kubectl`, perintah ini akan melakukan `API call` untuk perintah 
+yang kamu berikan. Kamu juga dapat menggunakan API Kubernetes secara langsung pada program yang kamu miliki 
+menggunakan salah satu [`library` klien](/docs/reference/using-api/client-libraries/) yang disediakan.
+
+### `Spec` dan Status Objek
+
+Setiap objek Kubernetes memiliki `field` berantai yang mengatur konfigurasi sebuah `object`:
+`spec` dan `status`. `Spec`, merupakan `field` yang harus kamu sediakan, `field` ini mendeskripsikan 
+`state` yang kamu inginkan untuk objek tersebut--karakteristik dari objek yang kamu miliki. 
+Statue mendeskripsikan `state` yang sebenarnya dari sebuah objek, dan hal ini disediakan dan selalu diubah oleh 
+sistem Kubernetes. Setiap saat, `Control Plane` Kubernetes selalu memantau apakah `state` aktual sudah sesuai dengan 
+`state` yang diinginkan.
+
+Sebagai contoh, `Deployment` merupakan sebuah objek yang merepresentasikan sebuah aplikasi yang dijalankan di kluster kamu. 
+Ketika kamu membuat sebuah `Deployment`, kamu bisa saja memberikan `spec` bagi `Deployment` untuk memberikan spesifikasi 
+berapa banyak `replica` yang kamu inginkan. Sistem Kubernetes kemudian akan membaca konfigurasi yang kamu berikan 
+dan mengaktifkan tiga buah `instance` untuk aplikasi yang kamu inginkan--mengubah status yang ada saat ini agar sesuai dengan apa yang kamu inginkan. 
```

Comment:
we should standardize between instance vs instans. `instans` was used in your other PR #14117